be/src/olap/txn_manager.cpp

```diff
@@ -76,6 +76,33 @@ TxnManager::TxnManager() {
     }
 }
 
+OLAPStatus TxnManager::prepare_txn(TPartitionId partition_id, const TabletSharedPtr& tablet, TTransactionId transaction_id, 
```

Comment:
1. May two prepare_txn() merged into one mothod?
2. partition_id can also get from tablet.

Reply:
1. It is very difficult to create tablet object during unit test and some times they are useful for example during load.
2. In the past, partition id not in tablet. so that I keep partition id as a parameter.